pkg/skaffold/build/ko/build.go

```diff
@@ -52,7 +52,13 @@ func (b *Builder) Build(ctx context.Context, out io.Writer, a *latestV1.Artifact
 	if err != nil {
 		return "", fmt.Errorf("could not build and publish ko image %q: %w", a.ImageName, err)
 	}
-	fmt.Fprintln(out, imageRef.Name())
+
+	outputRef := ref
+	if strings.HasSuffix(ref, ":latest") {
+		// handle tagPolicy sha256
+		outputRef = imageRef.Name()
+	}
+	fmt.Fprintln(out, outputRef)
```

Comment:
We're not outputting the image ref for the other builders.  Why is this necessary?

Reply:
Good question. `ko` itself prints it out at the end, and I've found that helpful, so this was just mimicking that behavior. And when using the Docker builder, the image name is printed at the end, even though Skaffold doesn't do it.

Reply:
ok makes sense